.travis.yml

```diff
@@ -1,16 +1,42 @@
 ---
-language: ruby
-bundler_args: --without development system_tests
-before_install: rm Gemfile.lock || true
 sudo: false
-rvm:
-  - 2.3.3
+dist: trusty
+language: ruby
+cache: bundler
+before_install:
+  - bundle -v
+  - rm -f Gemfile.lock
+  - gem update --system
+  - gem --version
+  - bundle -v
 script:
-  - bundle exec rake test
-  - bundle exec rubocop lib
-  - bundle exec puppet-lint --no-autoloader_layout-check manifests
+  - 'bundle exec rake $CHECK'
+bundler_args:  
+rvm:
+  - 2.4.4
 env:
-  - PUPPET_GEM_VERSION="~> 4.0"
```

Comment:
You previously ran unit tests against both Puppet 4 and 5 in travis, maybe something you want to add back in via sync.yml?

Reply:
@HelenCampbell Thanks for the valuable comment.It added both env puppet version 4 and 5 since default rvm version is added is 2.1.9 modified the travis .yaml file to use 2.3.3 and travis is running on both version now.